src/main/resources/jenkins/plugins/office365connector/Webhook/config.jelly

```diff
@@ -0,0 +1,65 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="URL" field="url">
+        <f:textbox />
+    </f:entry>
+    <f:entry title="Name" field="name">
+        <f:textbox />
+    </f:entry>
+
+    <f:advanced>
+        <f:section title="Build status">
+            <f:description>
+                This section defines for which build statuses the notification is sent.
+            </f:description>
+
+            <f:entry title="Notify Build Start" field="startNotification">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Aborted" field="notifyAborted">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Failure" field="notifyFailure">
+                <f:checkbox />
+            </f:entry>
+
```

Comment:
I think these needs to be changed... You updated in your previous PR... This is reverting those changes.

Reply:
I need to rebase onto master, that's all